heareval/tasks/config/speech_commands.py

```diff
@@ -2,18 +2,32 @@
 Configuration for the google speech commands task
 """
 
-from .dataset_config import *  # noqa: F403, F401
+from .dataset_config import PartitionedDatasetConfig, PartitionConfig
 
-TASKNAME = "speech_commands-v0.0.2"
-DOWNLOAD_URL = "http://download.tensorflow.org/data/speech_commands_v0.02.tar.gz"
-TEST_DOWNLOAD_URL = (
-    "http://download.tensorflow.org/data/speech_commands_test_set_v0.02.tar.gz"
-)
 
-SAMPLE_LENGTH_SECONDS = 1.0
-
-# Include the entire dataset, 85511 train files, 10102 validation files, and
-# 4890 testing samples
-MAX_TRAIN_FILES = None
-MAX_VAL_FILES = None
-MAX_TEST_FILES = None
+# TODO: Instead of having this as a python class -- this could be a json file and we could
+#   have a function that loads it and constructs the correct config object with that
+#   values from the json file. We could have a default json file that is loaded
+#   automatically -- and then if the user wants to adjust these values they could
+#   pass in an additional json config as a command line arg that would overwrite any
+#   default values that are specified.
```

Comment:
This comment is a bit confusing -- basic idea is that the classes in dataset_config.py will serve as the templates and the values for each dataset could be in json or something.